src/main/webapp/WEB-INF/tags/popupPriceHistory.tag

```diff
@@ -0,0 +1,104 @@
+<%@ tag trimDirectiveWhitespaces="true" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ attribute name="product" required="true" type="com.es.phoneshop.model.product.Product" %>
+
+<style>
```

Comment:
Please use css file in styles folder. You can use existing main.css or create your own and add it in master.tag